glossary.md

```diff
@@ -0,0 +1,22 @@
+# Glossary
+
+## Application
+
+A process tree launched inside a [container](#container).
+
+## Bundle
+
+A [directory structure](bundle.md) that is written ahead of time, distributed, and used to seed the runtime for creating [containers](#container) and launching [applications](#application).
```

Comment:
On Wed, Aug 12, 2015 at 09:42:27AM -0700, Brandon Philips wrote:
> > +## Bundle
> > +
> > +A [directory structure](bundle.md) that is written ahead of time, distributed, and used to seed the runtime for creating [containers](#container) and launching [applications](#application).
> 
> The intention is that a single bundle launches a single
> application. The bundle defines a single identity (bundle directory,
> cgroup hierarchy, etc) and a single lifecycle (hooks, etc).

Do we explicitly set that “single application” limit somewhere?  I
understand we have a default config location (config.json) for the
bundle [1], but I don't see why you couldn't store additional config
files in the bundle, and have optional runtime support for launching
from one of those additional configs [2].  Allowing the default config
filename to be overridden at runtime seems both useful and easy to
implement, so I don't see an upside to requiring a
single-application-per-bundle limit.

[1]: https://github.com/opencontainers/specs/blob/7e4a8779670cebc4125d352c0ca0c582474d5048/bundle.md#directory-layout
[2]: https://github.com/opencontainers/specs/pull/107/files#r36882200